download/alpha.html

```diff
@@ -28,15 +28,63 @@
         </div>
       </div>
       <div class="panel content" id="download-lin">
-        <h3>Mono for Linux is available for various distributions</h3>
-        <p>Please refer to the <a href="/docs/getting-started/install/linux">installation guide</a> for more information about how to install and configure your Mono environment.</p>
-        <h4>Xamarin packages</h4>
-        <div>
-          <p>These packages are built, tested and distributed by Xamarin. Use these if you want to use a stable, official and up-to-date version of Mono.</p>
-          <ul>
-            <li><a href="/docs/getting-started/install/linux/#debian-ubuntu-and-derivatives-alpha-channel">Debian, Ubuntu, and derivatives</a></li>
-            <li><a href="/docs/getting-started/install/linux/#centos-fedora-and-derivatives-alpha-channel">CentOS, Fedora, and derivatives</a></li>
-          </ul>
+        <dl id="mono-download-linux" class="tabs horizontal" data-tab data-options="deep_linking: true; scroll_to_content: false">
+          <dd class="active"><a href="#download-lin-debian"><i class="fa fa-debian"></i><img src="/images/lin_debian.png" />&nbsp;Debian</a></dd>
+          <dd><a href="#download-lin-ubuntu"><i class="fa fa-ubuntu"></i><img src="/images/lin_ubuntu.png" />&nbsp;Ubuntu</a></dd>
+          <dd><a href="#download-lin-raspbian"><i class="fa fa-raspbian"></i><img src="/images/lin_raspbian.png" />&nbsp;Raspbian</a></dd>
+          <dd><a href="#download-lin-centos"><i class="fa fa-centos"></i><img src="/images/lin_centos.png" />&nbsp;CentOS</a></dd>
```

Comment:
SVG instead of PNG (we can do it in a follow up cleanup if you don't have them handy).